hack/make_local.py

```diff
@@ -13,25 +13,6 @@ def get_env(line):
     return f'{env}={environ.get(env)}'
 
 
-def gen_local_deployments(outdir):
```

Comment:
Please remove the `import yaml` line

Reply:
Since I was not using linter for Python, I didn't catch it. Now, I installed a plugin. Thanks for this catch. Fixed.